docs/release-notes.md

```diff
@@ -45,6 +45,7 @@ nav_order: 9
 - Drop `LuksOption` and `RaidOption` types _(Go API for fcos 1.5.0+,
   flatcar 1.1.0+, openshift 4.14.0-experimental)_
 - Require Go 1.18+
+- Fail on invalid mointpoint configuration
```

Comment:
Lol this scared me for a second, make sure to fetch upstream and update your branch so that you dont have to move around your release notes.

Reply:
(thought I had not updated the release notes yet even though release had already happened)

Reply:
my bad should be fixed now!

Reply:
Nope, it isn't.